src/main/scala/dsls/regex/RegularExpression.scala

```diff
@@ -7,10 +8,48 @@ package dsls.regex
  * to *remove* anything that currently appears in the file.
  */
 
+ object regexImplicits {
+ 	implicit def charToLiteral(c : Char): RegularExpression =
+ 		Literal(c)
+ 	
+ 	implicit def stringToRegularExpression(s: String): RegularExpression = {
+ 		var firstConcat = Concat(Literal(s.charAt(0)),Literal(s.charAt(1)))
```

Comment:
This is much easier to do as `s.head ~ s.tail` (or something pretty close to that)